src/tests/class_tests/openms/executables.cmake

```diff
@@ -78,13 +78,11 @@ set(datastructures_executables_list
 set(metadata_executables_list
   AcquisitionInfo_test
   Acquisition_test
-  AnnotatedMSRun_test
```

Comment:
I think here you just replaced the file. See on the left side what was in the meanwhile added to the develop branch. This probably needs to be added back